.github/workflows/build.yml

```diff
@@ -147,11 +138,14 @@ jobs:
       # is the best source for possible updates. Once the UI is updated we will
       # most likely be able leverage the functionality of upload-artifact v2(+)
       # to upload a single file as an artifact.
-      - name: Unzip produced zip
-        run: unzip skeleton-aws-lambda.zip -d lambda_zip_contents
+      - name: Unzip generated zip
+        run: |
+          unzip ${{ github.event.repository.name }}.zip \
+            -d lambda_zip_contents
```

Comment:
I suggest using the long version of the command line flags here.

Reply:
There are no long versions of the command line flags for `zip` and `unzip`!  I shamefully withdraw my comment.

Reply:
Please see ec77abe for an expansion of the existing comment to at least describe what it is doing.